Base/QTApp/qSlicerApplicationHelper.cxx

```diff
@@ -311,15 +311,15 @@ bool qSlicerApplicationHelper::checkRenderingCapabilities()
     SHELLEXECUTEINFO shExecInfo;
     shExecInfo.cbSize = sizeof(SHELLEXECUTEINFO);
     shExecInfo.fMask = SEE_MASK_NOCLOSEPROCESS;
-    shExecInfo.hwnd = NULL;
+    shExecInfo.hwnd = nullptr;
```

Comment:
I suggest to check the documentation of `SHELLEXECUTEINFO` ?  Doing a test build would be sensible to.

Reply:
It is c++ code: https://docs.microsoft.com/en-us/windows/desktop/api/Shellapi/ns-shellapi-_shellexecuteinfoa

So it should be good.